tests/test_indieauth/test_relmeauth.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+class TestRelMeAuthLinkDiscovery:
+    @pytest.fixture
+    def target(self):
+        from indieweb_utils import get_valid_relmeauth_links
+
+        return get_valid_relmeauth_links
+
+    def test_rel_me_auth_link_discovery(self, target):
+        # NOTE: This test will take a few seconds as it executes multiple requests.
```

Comment:
We should pass in a hardcoded template with some hard-coded responses for each rel link and add those to responses objects so this test does not make use of the network. Similar as to how the [test_profile](https://github.com/capjamesg/indieweb-utils/blob/ca0ffe5f82e5fa24df61b9d5b58210f7bc9179ac/tests/test_indieauth/test_profile.py#L28) tests work.

Reply:
@capjamesg This should follow the other tests and use `responses` so it doesn't make a network call. 

```python
import responses

def test_rel_me_auth_link_discovery(self, target, index):
    url = "https://jamesg.blog"
    responses.add(responses.Response(method="GET", url=url, body=index))
    assert target(url) == "https://indieweb.social/@capjamesg"]
```
  
The `index` parameter in the function tells pytest to load the `index` fixture, which is located in [tests/conftest.py](https://github.com/capjamesg/indieweb-utils/blob/main/tests/conftest.py#L16), which simply reads `tests/fixtures/index.html` (which is a snapshot of the html on your index.html)